docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -122,6 +122,10 @@ application this will fail (as you cannot do blocking operations on the IO threa
 `Uni<SecurityIdentity> getDeferredIdentity();` method. You can then subscribe to the resulting `Uni` and will be notified
 when authentication is complete and the identity is available.
 
+NOTE: It's still possible to access the `SecurityIdentity` synchronously with `public SecurityIdentity getIdentity()`
+in the xref:resteasy-reactive.adoc[RESTEasy Reactive] from endpoints annotated with `@RolesAllowed`, `@Authenticated`,
+or with respective configuration authorization checks as authentication have already happened.
```

Comment:
```suggestion
or with respective configuration authorization checks as authentication have already happened.
```
```suggestion
or with respective configuration authorization checks as authentication has already happened.
```

Reply:
Not sure why the suggested change is shown twice, once without any actual suggested change. Only a minor typo fix is suggested

Reply:
Fixed the typo, thanks.